x-pack/test/apm_api_integration/tests/correlations/failed_transactions.spec.ts

```diff
@@ -203,10 +203,10 @@ export default function ApiTest({ getService }: FtrProviderContext) {
       expect(finalRawResponse?.overallHistogram?.length).to.be(101);
       expect(finalRawResponse?.fieldStats?.length).to.be(fieldsToSample.size);
 
-      expect(finalRawResponse?.failedTransactionsCorrelations?.length).to.eql(
-        30,
-        `Expected 30 identified correlations, got ${finalRawResponse?.failedTransactionsCorrelations?.length}.`
-      );
+      // expect(finalRawResponse?.failedTransactionsCorrelations?.length).to.eql(
+      //   30,
+      //   `Expected 30 identified correlations, got ${finalRawResponse?.failedTransactionsCorrelations?.length}.`
+      // );
```

Comment:
If this is no longer necessary I think can be deleted.

Reply:
I've uncommented it, good catch.